### ISSUE TITLE
LambdaRank output negative numbers 

### ISSUE THREAD

**Issue by areshand** [Microsoft/LightGBM#648]
## Environment info
Operating System: AWS box
CPU:
C++/Python/R version: c++ 11

The trained lambdarank model outputs negative score as below.
> 0.079443632115585708
> -0.025070996334226374
> -0.023198822324430338
> -0.62467651863218909
> -0.96513127586693603
> 0.28417701766747228

I used 0, 1, 2 in training data to indicate "irrelevant", "related", "highly relevant" as labels. I expected the prediction results should also fall into the range of [0, 2]. But the results contain negative numbers. How should I interpret the results? Is this a bug?

**Comment by guolinke** [comment]
This is normal. For the ranking, we didn't limit its output range. And you can sort the prediction score (in per query) to get the rank result.

**Comment by shashankg7** [comment]
@guolinke . Is there any range of scores for the ranking model? Upper limit and lower limit

**Comment by guolinke** [comment]
@shashankg7 I don't there is a bounded range, but they should be in the relatively small values.

**Comment by shashankg7** [comment]
Hi @guolinke, thanks for your prompt response. 

Since you mentioned relatively small ranges, I was wondering if I can apply a logistic function to ranking model's score? I want to get the output in the range [0,1].

**Comment by guolinke** [comment]
Yeah, I think you can
